website/install.html

```diff
@@ -100,6 +100,19 @@
       .os-column-table > .windows {
         grid-area: windows;
       }
+
+    @media (prefers-color-scheme: dark) {
+      a:link {color:#268BD2;}
+      a:visited {color:#6C71C4;}
+      body {background-color:  #002B36; color: white;}
+      td {color:black;}
+      .macos-logo-color {filter: invert(85%);} 
```

Comment:
Nit: Can we color-match the content logo from apple.com? https://www.apple.com/v/home/s/images/logos/watch-series-6/logo_hero__trsrpeu6koqy_medium_2x.png

Reply:
I color matched (it is pretty close to white) by figuring out the color of the logo which is #fafafa and then changing the svg file to the opposite hex value which is pretty close to black and then inverting that to get #fafafa.